dashboards/tests/test_grafana_sync.py

```
@@ -215,29 +273,50 @@ def test_cmd_push_uploads_changed_skips_conflicts(tmp_path, monkeypatch, capsys)
+        "def": {
+            "slug": "y",
+            "checksum": m.compute_checksum(conflict_original),
+            "remote_updated": "2026-01-01T00:00:00Z",
+            "folder_uid": "f1",
+        },
     }
     m.save_state(state)
 
     abc_call_count = {"n": 0}
+
     def fake_api_get(cfg, path):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `fake_api_get`

(ANN202)